git/essentials/features-iii/useful-commit-hooks.md

```diff
@@ -9,6 +9,12 @@ type: normal
 
 category: must-know
 
+aspects:
+  
+  - introduction
```

Comment:
I would remove `introduction` since it requires you to have knowledge of a few different components, and replace it with `workout`. This is also in-line with the tagging on `how-to-create-a-commit-hook`.